README.md

```diff
@@ -25,7 +25,7 @@ working, please import the neetoUI stylesheet to your main `scss` entry point.
 properly. Install the peer dependencies using the below command:
 
 ```
-yarn add react-toastify@9.0.1 formik@2.2.0 react-router-dom@5.2.0
+yarn add react-toastify@9.0.1 formik@2.2.0 react-router-dom@5.2.0 @bigbinary/neeto-icons antd i18next @emotion/is-prop-valid
```

Comment:
@anmol-kumar0815 _a 
1. Specify versions looking at the peerDependencies list.
2. Make sure this list contains absolute necessary requirements to run neetoui. i.e. If the basic functionality of neetoui would work without having `@emotion/is-prop-valid`, remove it from the list. You can ask people to optionally install `@emotion/is-prop-valid` if they want XYZ feature.

Reply:
@Amaljith-K _a, I have specified versions for peer dependencies except for neeto-icons because here we want to install the latest version.

If the changes look good to you, please resolve this conversation and merge this PR.